website/project/__init__.py

```diff
@@ -109,6 +109,35 @@ def new_collection(title, user):
 
     return node
 
+def new_public_files_collection(user):
+    """Create a new folder project.
+
+    :param str title: Node title
+    :param User user: User object
+    :return Node: Created node
+
+    """
+    existing_public_files_collections = Node.find(
+        Q('is_public_files_collection', 'eq', True) & Q('contributors', 'eq', user._id)
+    )
+
+    if existing_public_files_collections.count() > 0:
+        raise NodeStateError('Users may only have one public files collection')
+
+    title = 'Public Files'
```

Comment:
This also seems like an unnecessary variable declaration, as it's used on one line.